src/modules/dbus/dbusmodule.cpp

```diff
@@ -681,6 +681,29 @@ class Controller1 : public ObjectVTable<Controller1> {
 
     void setLogRule(const std::string &rule) { Log::setLogRule(rule); }
 
+    std::tuple<std::string, std::string, std::string, std::string, std::string,
+               std::string, std::string, bool, std::string>
+    currentInputMethodInfo() {
+        const auto &inputMethodManager = instance_->inputMethodManager();
+        const auto &group = inputMethodManager.currentGroup();
+
+        std::string currentIM = instance_->currentInputMethod();
+        const auto *entry = instance_->inputMethodManager().entry(currentIM);
+        if (entry == nullptr) {
+            return {"", "", "", "", "", "", "", false, ""};
```

Comment:
Simply {} should be enough

Also, previously I'd like it to be "sssssssbsa{sv}", while we have nothing for "a{sv}" (empty), we still want it in case we want to add anything in the future.